src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -426,6 +488,16 @@ ComputeTimeDerivative<Dim, EvolutionSystem, DgStepChoosers, LocalTimeStepping>::
                VarsPartialDerivatives::number_of_independent_components) *
               number_of_grid_points],
       VarsDivFluxes::number_of_independent_components * number_of_grid_points};
+  VarsFaceTemporaries face_temporaries{
```

Comment:
What about rather than creating a bigger face Variables here, just passing in the pointer given by `&buffer[(VarsTemporaries::number_of_independent_components +...` into the function, then creating the Variables in `internal_mortar_data_impl`? Then you need fewer template parameters in `internal_mortar_data_impl`, and you guarantee any weird contiguousness assumptions that the face vars might have assumed (I hope there are none, but :shrug:). Another data structure that avoids the tags and template parameter but keeps the size of the buffer is `gsl::span<double>`, so pass in a `gsl::not_null<gsl::span<double>*>`. I've done that and it works nicely. Simple function signature, then do non-owning Variables the way you do it already :D

Reply:
Went with the `gsl::span<double>` so I didn't have to work with raw pointers